src/components/admin/Players.vue

```diff
@@ -100,15 +102,22 @@ export default {
     });
   },
   methods: {
-    ...mapActions([
-      "apiFindPlayers",
-      "apiCreatePlayer",
-      "apiDeletePlayer",
-    ]),
+    ...mapActions(["apiFindPlayers", "apiCreatePlayer", "apiDeletePlayer"]),
     createPlayer: function () {
       this.apiCreatePlayer(
         this.$refs.playerCreationModal.newPlayerInfo.name
-      ).then(() => {
+      ).then((data) => {
+        if (data.createPlayer.id) {
+          this.$notify({
+            group: "success",
+            type: "success",
+            title: this.$t("playerCreated"),
+            text: this.$t("msgPlayerCreated", {
+              player: this.$refs.playerCreationModal.newPlayerInfo.name,
```

Comment:
The `createPlayer` mutation could return the `name` attribute. It could be easier than looking at the creation modal.